src/components/PublicHeader.spec.tsx

```diff
@@ -0,0 +1,33 @@
+import { mount } from '@cypress/react';
+
+import PublicHeader from './PublicHeader';
+
+describe('PublicHeader', () => {
+
+    const userData = {
+        avatar: 'https://api.zetk.in/v1/orgs/1/avatar',
```

Comment:
This is not an attribute that you can expect on user objects. You have to use the ID of the user in a fashion similar to how `OrgHeader` uses the ID of organizations.